src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -216,7 +216,15 @@ const VIEWPORT_TOP_OFFSET_DISTRACTION_FREE_MODE = 0;
                     content: locale.language,
                 },
                 ...extraConfig,
-            }).then((editor) => {
+            };
+
+            const event = new CustomEvent("ibexa-ckeditor:configure", {
+                detail: { container, config }
+            })
+
+            global.dispatchEvent(event);
```

Comment:
```suggestion
            doc.body.dispatchEvent(customEvent);
```
we are using `body` for such general events.